src/components/views/rooms/EventTile.tsx

```diff
@@ -757,6 +757,14 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
                 case EventShieldReason.MISMATCHED_SENDER_KEY:
                     shieldReasonMessage = _t("encryption|event_shield_reason_mismatched_sender_key");
                     break;
+
+                case EventShieldReason.SENT_IN_CLEAR:
+                    shieldReasonMessage = _t("common|unencrypted");
+                    break;
+
+                case EventShieldReason.VERIFICATION_VIOLATION:
+                    shieldReasonMessage = _t("encryption|event_shield_reason_verification_violation");
```

Comment:
I think it shoud match the existing copy for verification violation in invisible crypto? `Sender's verified identity has changed` as per `decryption_failure` > `sender_identity_previously_verified` ?
WDYT?

Reply:
Yup, good catch.  Fixed